examples/tutorials/translations/german/Part 01 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<h1>Tutorial: Die grundlegenden Werkzeuge des privatsphärenerhaltenden Deep Learning&nbsp;</h1><p> </p><p>Willkommen zum PySyfts Einführungs-Tutorial zum privatsphärenerhaltenden und dezentralisierten Deep Learning. Diese Reihe von Notebooks ist eine schrittweise Anleitung, mit der Sie die neuen Werkzeuge und Techniken kennenlernen können, die für das Deep Learning an geheimen oder privaten Daten und Modellen erforderlich sind, ohne sie dem Eigentum einer Instanz zu unterwerfen.</p><p>&nbsp;</p><p><strong>Was wir behandeln werden:</strong> Beachten Sie, dass wir nicht nur über das Dezentralisieren / Verschlüsseln von Daten sprechen, sondern auch darüber, wie PySyft verwendet werden kann, um das gesamte Ökosystem um Daten herum zu dezentralisieren, einschließlich der Datenbanken, in denen sich Daten befinden, gespeichert oder abgefragt werden, sowie die neuronalen Netzwerke, mit denen Informationen aus Daten extrahiert werden. Wenn neue Erweiterungen für PySyft erstellt werden, werden diese Notebooks um neue Tutorials erweitert, um die neuen Funktionen zu erläutern.&nbsp;</p><p>Autoren:&nbsp;</p><ul><li>Andrew Trask - Twitter:&nbsp;<a href="https://twitter.com/iamtrask" target="_blank">@iamtrask</a></li></ul><p> </p><p>Übersetzer:&nbsp;</p><ul><li>Vineet Jain - Github:&nbsp;<a href="https://github.com/vineetjai" target="_blank">@vineetjai</a></li><li>Georgios Kaissis - Twitter: <a href="https://twitter.com/GKaissis" target="_blank">@gkaissis</a></li></ul><h2>Gliederung:&nbsp;</h2><ul><li>Teil 1: Die grundlegenden Werkzeuge des privaten Deep Learning&nbsp;</li></ul><h2>Warum dieses Tutorial?&nbsp;</h2><p> </p><p><strong>1) Wettbewerbsvorteil für Ihre Karierre</strong> - In den letzten 20 Jahren hat die digitale Revolution die Daten in immer größeren Mengen zugänglicher gemacht, da analoge Prozesse digitalisiert wurden. Mit neuen Regelungen wie z.B. der <a href="https://eugdpr.org/" target="_blank">Datenschutzgrundverordnung</a>, stehen Unternehmen zunehmend unter Einschränkungen Ihrer Freiheit bei der Verwendung - und vor allem bei der Analyse - persönlicher Informationen. <strong> Fazit: </strong> Datenwissenschafftler werden mit "Old School" -Werkzeugen nicht mehr auf so viele Daten zugreifen können, aber durch das Erlernen der Tools des privatspährenerhaltenden Deep Learning können SIE dieser Kurve voraus sein und einen Wettbewerbsvorteil für Ihre Karriere erzielen.&nbsp;</p><p><strong> 2) Unternehmerische Möglichkeiten </strong> - Es gibt eine ganze Reihe von Problemen in der Gesellschaft, die Deep Learning lösen kann, aber viele der wichtigsten wurden noch nicht untersucht, da dies den Zugriff auf sensible Informationen über Menschen erfordern würde (erwägen Sie die Verwendung von Deep Learning, um Menschen mit mentalen oder Beziehungsproblemen zu helfen!). Das Erlernen von privatspährenerhaltendem Deep Learning eröffnet Ihnen somit eine Vielzahl neuer Startmöglichkeiten, die anderen ohne diese Tools bisher nicht zur Verfügung standen.&nbsp;</p><p><strong>3) Sozialer Fortschritt</strong> - Deep Learning kann verwendet werden, um eine Vielzahl von Problemen in der realen Welt zu lösen, aber Deep Learning an <em>persönlichen Informationen</em> ist Deep Learning über Menschen, <em>für Menschen</em>. Das Erlernen des Deep Learning für Daten, die Sie nicht besitzen, ist mehr als eine berufliche oder unternehmerische Gelegenheit. Es ist die Gelegenheit, einige der persönlichsten und wichtigsten Probleme im Leben der Menschen zu lösen - und dies in großem Maßstab.&nbsp;</p><h2>Wie bekomme ich Bonuspunkte?&nbsp;</h2><ul><li>Gib PySyft einen Stern auf GitHub! -&nbsp;<a href="https://github.com/OpenMined/PySyft" target="_blank">https://github.com/OpenMined/PySyft</a></li><li>Mach ein Youtube-Video, in dem dieses Notebook's unterrichtet wird!&nbsp;</li></ul><p>... Ok, los geht's!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Also, Mach ein Youtube-Video, in dem dieses Notebook's unterrichtet wird!&nbsp; (line -2) should be <span class="ql-cursor">﻿</span>Mach ein Youtube-Video, in dem dieses Notebook unterrichtet wird!&nbsp;</p>